tests/E2E/GzipEncodingTest.php

```diff
@@ -30,7 +28,7 @@ public function testContentEncodingHeader()
 
         $this->assertEquals('gzip', $logEntry->getRequest()->getHeaderLine('Accept-Encoding'));
 
-        // Need to check 'x-encoded-content-encoding' as curl is automatically decompressing the response
-        $this->assertEquals('gzip', $logEntry->getResponse()->getHeaderLine('x-encoded-content-encoding'));
+        // Need to check 'X-Encoded-Content-Encoding' as curl is automatically decompressing the response
+        $this->assertEquals('gzip', $logEntry->getResponse()->getHeaderLine('X-Encoded-Content-Encoding'));
```

Comment:
Yes, Guzzle lowercases everything, so this is just for consistency with routine use.